packages/credential-provider-sso/README.md

```diff
@@ -0,0 +1,94 @@
+# @aws-sdk/credential-provider-sso
+
+[![NPM version](https://img.shields.io/npm/v/@aws-sdk/credential-provider-sso/latest.svg)](https://www.npmjs.com/package/@aws-sdk/credential-provider-sso)
+[![NPM downloads](https://img.shields.io/npm/dm/@aws-sdk/credential-provider-sso.svg)](https://www.npmjs.com/package/@aws-sdk/credential-provider-sso)
+
+## AWS Credential Provider for Node.JS - AWS Single Sign-On(SSO)
+
+This module provides a function, `fromSSO` that will create `CredentialProvider`
+functions that read from [AWS SDKs and Tools shared configuration and credentials files](https://docs.aws.amazon.com/credref/latest/refdocs/creds-config-files.html)(Profile appears
+in the credentials file will be given precedence over the profile found in the
+config file). This provider will load the _resolved_ access token on local disk,
+and then request temporary AWS credentials. For the guidance over AWS Single
+Sign-On service, please refer to [the service document](https://aws.amazon.com/single-sign-on/)
+
+## Supported configuration
+
+You may customize how credentials are resolved by providing an options hash to
+the `fromSSO` factory function. The following options are supported:
+
+- `profile` - The configuration profile to use. If not specified, the provider
+  will use the value in the `AWS_PROFILE` environment variable or a default of
+  `default`.
+- `filepath` - The path to the shared credentials file. If not specified, the
+  provider will use the value in the `AWS_SHARED_CREDENTIALS_FILE` environment
+  variable or a default of `~/.aws/credentials`.
+- `configFilepath` - The path to the shared config file. If not specified, the
+  provider will use the value in the `AWS_CONFIG_FILE` environment variable or a
+  default of `~/.aws/config`.
```

Comment:
nit: use `by default` instead of `a default value of`.
```suggestion
- `profile` - The configuration profile to use. If not specified, the provider
  will use the value in the `AWS_PROFILE` environment variable or `default` by
  default.
- `filepath` - The path to the shared credentials file. If not specified, the
  provider will use the value in the `AWS_SHARED_CREDENTIALS_FILE` environment
  variable or `~/.aws/credentials` by default.
- `configFilepath` - The path to the shared config file. If not specified, the
  provider will use the value in the `AWS_CONFIG_FILE` environment variable or
  `~/.aws/config` by default.
```

Reply:
I copied this from the default provider: https://github.com/aws/aws-sdk-js-v3/blob/7b2794444c47bf335e156a7f708280c98f9189a3/packages/credential-provider-node/README.md#L35-L40

But happy to change.